### PR TITLE
[Kotlin Migration] SchedTimingToday 

### PR DESCRIPTION
**Pull request by codingtosh**
[Kotlin Migration] SchedTimingToday 
package com.ichi2.libanki.backend.model

### CODE REVIEW

**Review by david-allison-1 — CHANGES_REQUESTED** [review]

**Review by david-allison-1 — CHANGES_REQUESTED** [review]
one nit, then good to go, no need for a `cleanup` here

File: AnkiDroid/src/main/java/com/ichi2/libanki/backend/model/SchedTimingToday.kt [hunk 1]
```diff
@@ -13,16 +13,15 @@
...
 
-public interface SchedTimingToday {
-    /** The number of days that have passed since the collection was created.<br/>
-     *  uint32 upstream */
-    int days_elapsed();
     /**
-     * Timestamp of the next day rollover.<br/>
-     * int64 upstream */
-    long next_day_at();
+     * Timestamp of the next day rollover.<br></br>
```
[the remark this answers is not in the document]
Comment (codingtosh) [reply AnkiDroid/src/main/java/com/ichi2/libanki/backend/model/SchedTimingToday.kt]: Done, PTAL.

**Review by david-allison-1 — APPROVED** [review]

File: AnkiDroid/src/main/java/com/ichi2/libanki/backend/model/SchedTimingToday.kt [hunk 2]
```diff
@@ -13,16 +13,19 @@
...
 
-public interface SchedTimingToday {
-    /** The number of days that have passed since the collection was created.<br/>
-     *  uint32 upstream */
-    int days_elapsed();
     /**
-     * Timestamp of the next day rollover.<br/>
-     * int64 upstream */
-    long next_day_at();
+     * Timestamp of the next day rollover.
```
Comment (Akshay0701) [inline AnkiDroid/src/main/java/com/ichi2/libanki/backend/model/SchedTimingToday.kt:26]: `<br/>`is missing

File: AnkiDroid/src/main/java/com/ichi2/libanki/backend/model/SchedTimingToday.kt [hunk 3]
```diff
@@ -13,16 +13,19 @@
...
+    /** The number of days that have passed since the collection was created.
+     *
+     * uint32 upstream
+     * */
+    fun days_elapsed(): Int
 
-public interface SchedTimingToday {
-    /** The number of days that have passed since the collection was created.<br/>
-     *  uint32 upstream */
-    int days_elapsed();
```
Comment (Akshay0701) [inline AnkiDroid/src/main/java/com/ichi2/libanki/backend/model/SchedTimingToday.kt:22]: `<br/>`is missing

**Review by Akshay0701 — CHANGES_REQUESTED** [review]
just two `<br>` missing, otherwise LGTM

File: AnkiDroid/src/main/java/com/ichi2/libanki/backend/model/SchedTimingToday.kt (hunk 2 above)
Comment (mikehardy) [reply AnkiDroid/src/main/java/com/ichi2/libanki/backend/model/SchedTimingToday.kt:26]: @Akshay0701 the br comment seems valid - if you require changes in a PR can you add the label "waiting for feedback"? When I work on AnkiDroid I move really quickly through PR searches defined by the labels, and if it has that label then I skip it until the label is removed. Helps me move faster :-)

Comment (mikehardy) [reply AnkiDroid/src/main/java/com/ichi2/libanki/backend/model/SchedTimingToday.kt:26]: @Akshay0701 the br comment seems valid - if you require changes in a PR can you add the label "waiting for feedback"? When I work on AnkiDroid I move really quickly through PR searches defined by the labels, and if it has that label then I skip it until the label is removed. Helps me move faster :-)

Comment (codingtosh) [reply AnkiDroid/src/main/java/com/ichi2/libanki/backend/model/SchedTimingToday.kt:26]: @Akshay0701 The `<br>` tags  were replaced with a newline as per David's suggestion, PTAL
https://github.com/ankidroid/Anki-Android/pull/10199#discussion_r784472143

Comment (mikehardy) [reply AnkiDroid/src/main/java/com/ichi2/libanki/backend/model/SchedTimingToday.kt:26]: Ah! I'm rolling through the queue now and re-checked this based on the update notification. I can see what you mean, after showing the resolved conversations above. Marking for merge. Thanks to both of you for checking + working through these migrations, much appreciated

**Review by mikehardy — APPROVED** [review]

**Pull request merged**